### DISCUSSION TITLE
#13

### DISCUSSION THREAD

**Discussion by Ad-closeNN** [MFn233/PCL-Mainpage#15] [一般]
#13

**Comment by Ad-closeNN** [comment]
#13 在这继续聊

**Comment by SqrtTan** [comment]
不出意外肯定是关了（毕竟bbs是公益站）

**Reply by SqrtTan** [reply]
妈耶我discord上不去（steam++好像不提供discord加速）

**Reply by Ad-closeNN** [reply]
> 妈耶我discord上不去（steam++好像不提供discord加速）

对去年就没了这个加速

**Comment by Ad-closeNN** [comment]
bbs有java下吗，如果有的话以后只能使用[高校源](https://github.com/Hex-Dragon/PCL2/issues/3374)来快速下载OpenJDK了

**Reply by SqrtTan** [reply]
不清楚，我平时都开尽量官方源，浙江网络不错

**Reply by Ad-closeNN** [reply]
广西电信没封

**Comment by Ad-closeNN** [comment]
BMCLAPI肯定是顶不住的，净网也不会开锁的

**Reply by SqrtTan** [reply]
所以说让有能力的人用OpenBMCLAPI部署啊
